sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -117,62 +69,113 @@ cmake -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="X86" \
 -DLLVM_ENABLE_PROJECTS="clang;llvm-spirv;sycl" \
 -DLLVM_EXTERNAL_SYCL_SOURCE_DIR=$SYCL_HOME/llvm/sycl \
 -DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR=$SYCL_HOME/llvm/llvm-spirv \
--DOpenCL_INCLUDE_DIR="$OPENCL_HEADERS" -DOpenCL_LIBRARY="$ICD_LIB" \
 $SYCL_HOME/llvm/llvm
 
 make -j`nproc` sycl-toolchain
```

Comment:
why we have make on Linux and ninja on linux. lets use Ninja everywhere as it expected to be more effective

Reply:
I think @v-klochkov answered this question already in https://github.com/intel/llvm/pull/494. IIRC, `ninja` is installed with Visual Studio.
`make` is more ubiquitous on Linux, but `ninja` can be also used.